client/state/purchases/selectors.js

```diff
@@ -1,7 +1,20 @@
-// External dependencies
-import assign from 'lodash/assign';
-import find from 'lodash/find';
+/**
+ * Return user's purchases from state object
```

Comment:
Are we really only returning user's purchases here? What about site's purchases? It looks like this can be any of them.